### DISCUSSION TITLE
ArgoCD doesn't sync to OCI Helm chart (ACR). Is that correct?

### DISCUSSION THREAD

**Discussion by hoanglhifi** [argoproj/argo-cd#8616] [General]
Situation:  
My ArgoCD app is created and synced to Azure OCI Registry Helm chart. (ex: Chart version 1.0.0).  
Synchronize was successful at the first time.  
But after I delete the Helm Chart version 1.0.0 from Azure OCI registry (ACR), ArgoCD still show that sync successful when I click on Sync button. (I expect the error will happen)  

Question:  
- ArgoCD stored Helm chart v1.0.0 and not pull it again when I click Sync button, is that right?  
- If I push some changes overwrite the Helm Chart version 1.0.0 on OCI, How can I force ArgoCD pull the charts once more time?

**Comment by crenshaw-dev** [comment]
Argo CD caches manifests. If the chart version hasn't changed, it has no reason to fetch the chart. If you hit "hard refresh," Argo CD will ignore the cache and attempt to load from the registry.

**Reply by hoanglhifi** [reply]
exactly thank you

**Comment by ebuildy** [comment]
it could use the artefact digest as cache key?
